nautobot/extras/forms/forms.py

```diff
@@ -356,22 +362,15 @@ def _append_customfield_fields(self):
 
         # Append form fields
         for cf in CustomField.objects.filter(content_types=self.obj_type):
-            field_name = "cf_{}".format(cf.name)
+            field_name = "cf_{}".format(cf.slug)
             self.fields[field_name] = cf.to_form_field(for_csv_import=True)
 
             # Annotate the field in the list of CustomField form fields
             self.custom_fields.append(field_name)
 
 
 class CustomFieldBulkCreateForm(CustomFieldBulkEditForm):
-    """
-    Adaptation of CustomFieldBulkEditForm which uses prefixed field names
-    """
-
-    @staticmethod
-    def _get_field_name(name):
-        # Return a prefixed version of the name
-        return "cf_{}".format(name)
+    """No longer needed as a separate class - use CustomFieldBulkEditForm instead."""
```

Comment:
Raise a `DeprecationWarning` on instantiation perhaps?

Reply:
Will add. DeprecationWarning is silenced by default, and it'd be better if we could warn on class inheritance rather than subclass instantiation, but it's better than nothing.